.github/CODEOWNERS

```
@@ -7,10 +7,10 @@
 * @newrelic/k8s-agents
 
 # APM team owned directories
-/src/apm/dotnet.go @newrelic/dotnet
-/src/apm/golang.go @newrelic/go-agent
-/src/apm/javaagent.go @newrelic/java-agent
-/src/apm/nodejs.go @newrelic/node-js-agent
-/src/apm/php.go @newrelic/php-agent
-/src/apm/python.go @newrelic/python
-/src/apm/ruby.go @newrelic/ruby-agent
+/src/apm/dotnet*.go @newrelic/dotnet
+/src/apm/golang*.go @newrelic/go
+/src/apm/java*.go @newrelic/java
+/src/apm/nodejs*.go @newrelic/node
+/src/apm/php*.go @newrelic/php-agent
```

```suggestion
/src/apm/php*.go @newrelic/phpc 
```
This needs to specify `phpc` instead of `php-agent` as the team name